openedx/features/edly/models.py

```diff
@@ -1,9 +1,11 @@
+from django.contrib.auth.models import User
 from django.contrib.sites.models import Site
 from django.core.validators import RegexValidator
 from django.db import models
 from model_utils.models import TimeStampedModel
```

Comment:
separate django imports with local imports as done in edX.

Reply:
`isort` is reconverting it. I am now changing it manually.